### PR TITLE
chore(Wizard): wrap Wizard modal example in page

### LINKED ISSUE #7403: Wizard demos - add standard page layout to full page

### PR DESCRIPTION
**Pull request by jenny-s51**
**What**: Closes #7403

### CODE REVIEW

**Comment by nicolethoen** [comment]
quick question - did you opt not to use the dashboard wrapper for any particular reason? If there are someday updates to the dashboard wrapper or the 'standard masthead/page layout' we will have a lot more places to update the more demos we build without the dashboard wrapper.

**Review by jenny-s51 — COMMENTED** [review]
Thank you @nicolethoen - you make a good point! Initially I was running into some issues using the `DashboardWrapper` here but I figured out what was going on after some investigating! Updated both of these demos to use the `DashboardWrapper` now 👍

File: packages/react-core/src/demos/Wizard/WizardDemo.md [hunk 1]
```diff
@@ -3,41 +3,85 @@ id: Wizard
...
+  Breadcrumb,
+  BreadcrumbItem,
+  Card,
+  CardBody,
+  Gallery,
+  GalleryItem,
+  Nav,
+  NavItem,
+  NavList,
+  Page,
```
Comment (nicolethoen) [inline packages/react-core/src/demos/Wizard/WizardDemo.md:27]: Are some of these unused imports, now?

**Review by nicolethoen — APPROVED** [review]

**Review by mmenestr — APPROVED** [review]
LGTM

**Review by mattnolting — APPROVED** [review]
LGTM! 👍

**Pull request merged**